app/main/db/queries.py

```diff
@@ -181,17 +183,16 @@ def _build_transferring_body_view_query(transferring_body_id):
     return query
 
 
-def _build_series_view_query(series_id):
-    query = (
+def _build_series_view_query(series_id, filters, sorting_orders):
+    sub_query = (
         db.session.query(
             Body.BodyId.label("transferring_body_id"),
             Body.Name.label("transferring_body"),
             Series.SeriesId.label("series_id"),
             Series.Name.label("series"),
-            func.to_char(
-                func.cast(func.max(Consignment.TransferCompleteDatetime), DATE),
-                current_app.config["DEFAULT_DATE_FORMAT"],
-            ).label("last_record_transferred"),
+            func.max(Consignment.TransferCompleteDatetime).label(
```

Comment:
without this change does the sorting not work as expected due to the format of the date?

Reply:
Yes, with DD/MM/YYYY format does not allow correct sorting as postgres DB support yyyy-mm-dd as a default format